DEV-9/RandomReplacementPartsOfLines/RandomNumbersBuilder.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace RandomReplacementPartsOfLines
+{
+    class RandomNumbersBuilder
+    {
+        Random randomNumber = new Random();
+
+        // Method returns the index of beginning
+        // of the replaced subline.
+        // Parameter 0 in method Next() means the first
+        // possible value of index.
+        public int BuildBeginningOfReplacedSubline(string initialLine)
+        {
+            int indexOfBeginningOfReplacedSubline = randomNumber.Next(0, initialLine.Length);
+            return indexOfBeginningOfReplacedSubline;
+        }
+
+        // Method returns the length
+        // of the replaced subline.
+        // Parameter 1 in method Next() means the minimal
+        // possible value of the length.       
+        public int BuildLengthOfReplacedSubline(string initialLine, int beginningOfReplacedSubline)
```

Comment:
Why do you implement two methods with similar functionality?
BuildLengthOfReplacedSubline
BuildLengthOfReplacingSubline

BuildBeginningOfReplacingSubline
BuildBeginningOfReplacedSubline